test/csx.coffee

```diff
@@ -69,6 +69,30 @@ test 'attribute without value', ->
     <div checked x="hello" />;
   '''
 
+test 'reserved-word attribute without value', ->
+  eqJS '''
+    <div static x="hello" />
+  ''', '''
+    <div static x="hello" />;
+  '''
+
+test 'reserved-word attribute with value', ->
+  eqJS '''
+    <div static="yes" x="hello" />
+  ''', '''
+    <div static="yes" x="hello" />;
+  '''
+
+test 'attribute with namespace', ->
+  eqJS '''
+    <image xlink:href="data:image/png" />
+  ''', '''
+    <image xlink:href="data:image/png" />;
+  '''
+
+test 'attribute with double namespace disallowed', ->
+  throws -> CoffeeScript.compile '<image xlink:href:tag="data:image/png" />'
+
```

Comment:
Tests added. It's odd that Babel allows `<a:b.c />` to compile. It compiles to `react.createElement("a:b".c)` which I can't imagine being useful (it typically results in an `undefined` tag).  I'm not a React/JSX expert, but my impression is that `a:b` is useful for making literal output tags with namespaces, and `b.c` is useful when `b` is a virtual tag implemented by a JavaScript function or class.  Unless I'm missing something, this makes them useless together as `a:b.c`, because `a:b` will never be a JavaScript function or class (being an invalid JavaScript identifier).  This explains why it's forbidden in the spec (but not why it's supported by Babel).

Reply:
Ok, that makes sense to me that `<a:b.c>` probably isn't a useful construct

So I think it's fine to not support it